src/pdf/DayOverview/DayOverviewPdfButton.tsx

```diff
@@ -75,24 +75,24 @@ const addCongestionChargeDetailsForDayOverview = async (
     const postcodes: (string | null)[] = [];
 
     for (const parcel of parcels) {
-        if (parcel.client?.address_postcode) {
-            postcodes.push(parcel.client?.address_postcode);
-        }
+        parcel.client?.address_postcode
+            ? postcodes.push(parcel.client?.address_postcode)
+            : postcodes.push(null);
```

Comment:
Do we definitely want to push null postcodes? can you explain why please?

Reply:
reply in my recent self-review